### ISSUE TITLE
[compat/request] Compatibility to Create Fluids

### ISSUE THREAD

**Issue by noawx** [Tiviacz1337/Travelers-Backpack#884]
### **Is your feature request related to a problem? Please describe.**
I like to play with both Create and Traveler's Backpack, but there's a problem. Create and Traveler's Backpack use different milk and potions libs.
So when I try to use pipes to insert potions or milk, I cannot drink them from the hose.
And if I try to use pipes to remove potions or milk, I cannot use them in create recipes.

### **Describe the solution you'd like**
Basically that the potions and milk from Create can be drank from hoses, and when emptying a milk bucket or potion bottle into the tanks, it turns into the create version of the fluids. (If create is enabled)
This would allow for automation, inserting and removing potions and milk from the Backpack.

### **Describe alternatives you've considered**
Manually putting the milk buckets and potion bottles into/out of the backpack.

**Comment by Tiviacz1337** [comment]
forge/fabric?

**Comment by noawx** [comment]
I'm on Quilt. (so fabric)
